napari/_qt/qt_main_window.py

```diff
@@ -151,6 +151,28 @@ def __init__(self, viewer, *, show: bool = True):
         else:
             self._debug_menu = None
 
+        from .qt_viewer import shortcut_manager
+
+        shortcut_manager.bind_shortcut(
+            'toggle_console_visibility', 'Ctrl+Shift+C'
+        )
+        for name, (command, desc, s) in shortcut_manager._commands.items():
+            qa = QAction(s)
+            qa.setText(name)
+            qa.setStatusTip(desc)
+            qa.triggered.connect(command)
+            sht = shortcut_manager._shortcuts.get(name, None)
+            if sht is not None:
+                qa.setShortcut(sht)
+            self.window_menu.addAction(qa)
```

Comment:
This is currently a bit redundant and the `toggleViewAction` is already added on line 724 of this file for all qwidget pannels, but that's a proof of concept; the toggle view action also will have a checkbox in the menu, which this does not but this is fixable.

Reply:
Is this `name` already localized or is it more like the action Unique ID?